### PR TITLE
[feature] Add tests for the api end-points

### PR DESCRIPTION
**Pull request by kwanj-k**

### CODE REVIEW

File: api/app.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+from flask import Flask, request, jsonify, make_response
+import json 
+
+app = Flask(__name__)
+
+
+
+if __name__ == '__main__':
+    unittest.main()
```
Comment (kwahalf) [inline api/app.py]: add a new line at the end of every file

File: api/tests/test_meals.py [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+        result = self.app.get('/meals/1')
+
+        self.assertEqual(result.status_code, 404)
+
+       
+
+
+if __name__ == '__main__':
+    unittest.main()
+
```
Comment (kwahalf) [inline api/tests/test_meals.py]: same here

File: api/tests/test_auth.py [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+           result['message'], "User already exists. Please login.")
+
+
+
+
+
+
+if __name__ == '__main__':
+    unittest.main()
+
```
Comment (kwahalf) [inline api/tests/test_auth.py]: add a new line at the end of every file

File: api/tests/test_menu.py [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+        res = self.app.get('/api/v1/menus')
+
+        self.assertEqual(res.status_code, 200)
+
+       
+
+
+if __name__ == '__main__':
+    unittest.main()
+
```
Comment (kwahalf) [inline api/tests/test_menu.py]: same here

File: api/tests/test_orders.py [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+
+        res = self._and_createapp.get(
+                '/api/v1/oreders/1'
+            )
+        self.assertIn('badgia', str(res.data))
+
+
+if __name__ == '__main__':
+    unittest.main()
+
```
Comment (kwahalf) [inline api/tests/test_orders.py]: same here

File: api/tests/test_menu.py [hunk 6]
```diff
@@ -0,0 +1,46 @@
+from app import app
+import unittest
+import json
+
+
+
```
Comment (kwahalf) [inline api/tests/test_menu.py]: too many new lines use pepe8 style of formating

File: api/tests/test_auth.py [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+    def setUp(self):
+        app.testing = True
+        self.app = app.test_client()
+        self.data = {"username":"zeus",
+        				 "email":"email@gmail.com",
+        				 "password":"4084"}
+       
+
+    def test_signup_and_login(self):
```
Comment (kwahalf) [inline api/tests/test_auth.py]: test a single endpoint at a time ie, test register separately then test login

File: api/tests/test_meals.py [hunk 8]
```diff
@@ -0,0 +1,76 @@
...
+        self.app = app.test_client()
+
+        self.data = {
+                        "id":1,
+                        "meal_name":"lunch"
+        				 }
+       
+
+    def test_get_and_create_meals(self):
```
Comment (kwahalf) [inline api/tests/test_meals.py]: same here, test get meals separately and create meals separately

Comment (kwanj-k) [reply api/tests/test_meals.py]: noted,I am working on it

**Pull request merged**